### PR TITLE
Update the OWIN doc and samples for 1.0.0

### PR DESCRIPTION
**Pull request by Tratcher**
@Rick-Anderson

### CODE REVIEW

File: aspnet/fundamentals/owin.rst [hunk 1]
```diff
@@ -48,53 +48,57 @@ You can of course configure other actions to take place within the OWIN pipeline
...
+``IServer`` is an interface that requires an ``Features`` property and a ``Start`` method.
 
-``Initialize`` is responsible for configuring the server, which in this case is done through a series of fluent API calls that hard code the server to listen for requests (to any IP address) on port 5000. Note that the final line of the fluent configuration of the ``builder`` variable specifies that requests will be handled by the private method ``HandleRequest``.
+``Start`` responsible for configuring and starting the server, which in this case is done through a series of fluent API calls that set addresses parsed from the IServerAddressesFeature. Note that the fluent configuration of the ``_builder`` variable specifies that requests will be handled by the ``appFunc`` defined earlier in the method. This ``Func`` is called on each request to process incoming requests.
```
Comment (Rick-Anderson) [inline aspnet/fundamentals/owin.rst]: >This ``Func`` is called on each request to process incoming requests. 

Is there anyway to simplify that?

Can we just say 
This ``Func`` is called on each request.

File: aspnet/fundamentals/owin.rst [hunk 2]
```diff
@@ -55,58 +56,53 @@ You can of course configure other actions to take place within the OWIN pipeline
...
 
 .. image:: owin/_static/websocket-test.png
 
 
 OWIN keys
 ---------
 
-OWIN depends heavily on an ``IDictionary<string,object>`` used to communicate information throughout an HTTP Request/Response exchange. ASP.NET Core implements all of the required and optional keys outlined in the OWIN specification, as well as some of its own. Note that any keys not required in the OWIN specification are optional and may only be used in some scenarios. When working with OWIN keys, it's a good idea to review the list of `OWIN Key Guidelines and Common Keys <http://owin.org/spec/spec/CommonKeys.html>`_
+OWIN depends on an ``IDictionary<string,object>`` object to communicate information throughout an HTTP Request/Response exchange. ASP.NET Core implements all of the keys outlined in the OWIN specification, and some ASP.NET Core OWIN keys. Any keys not required in the OWIN specification are optional and may only be used in some scenarios. See `OWIN Key Guidelines and Common Keys <http://owin.org/spec/spec/CommonKeys.html>`_
```
Comment (Rick-Anderson) [inline aspnet/fundamentals/owin.rst]: >Any keys not required in the OWIN specification are optional and may only be used in some scenarios.

Can you write that less nebulously?  That's totally useless. What's the limitation? Portability?

**Comment by Rick-Anderson** [comment]
@Tratcher  please check my changes. Nice work!

**Comment by Rick-Anderson** [comment]
@Tratcher  I can make any updates you need. Do you want me to add you to the authors list?

File: aspnet/fundamentals/owin.rst (hunk 2 above)
Comment (Tratcher) [inline aspnet/fundamentals/owin.rst]: "OWIN depends on an ``IDictionary<string,object>`` object to communicate information throughout an HTTP Request/Response exchange. ASP.NET Core implements the keys listed below. See the `primary specification, extensions <http://owin.org/#spec>`_, and `OWIN Key Guidelines and Common Keys <http://owin.org/spec/spec/CommonKeys.html>`_"

**Comment by Tratcher** [comment]
Thanks for making the updates.
No need to add me to the authors.

**Pull request merged**